### PR TITLE
Add new function: skip vlan, scan vlan, and scan trunk port

### PR DESCRIPTION
**Pull request by xxconn**
Add new function:

In site, add skip vlan, and scan vlan

In device, add scan-trunk-port for VM.

### CODE REVIEW

File: setup.php [hunk 1]
```diff
@@ -1464,6 +1485,26 @@ function mactrack_config_form () {
 		'value' => '|arg1:site_info|',
 		'max_length' => '255'
 		),
+	'skip_vlans' => array(
+		'method' => 'textarea',
+		'friendly_name' => 'VLAN ID to Skipped Scan',
+		'class' => 'textAreaNotes',
+		'textarea_rows' => '5',
+		'textarea_cols' => '80',
+		'description' => 'VLAN ID always forced to be skipped scanning',
```
Comment (TheWitness) [inline setup.php]: Should indicate is column separated or space in descriptions.  I think in MakTrack, it's generally comma. Double check though.

File: setup.php [hunk 2]
```diff
@@ -1322,6 +1332,17 @@ function mactrack_config_form () {
 		'textarea_cols' => '80',
 		'max_length' => '255'
 		),
+	'scan_trunk_port' => array(
+		'method' => 'textarea',
+		'friendly_name' => 'Trunk Ports to Scan',
+		'description' => 'Some VMware network port will be configured with 802.1Q',
```
Comment (TheWitness) [inline setup.php]: Same comment here.  You could say something like "A comma delimited is of ..."

File: setup.php [hunk 3]
```diff
@@ -1464,6 +1485,26 @@ function mactrack_config_form () {
...
+		'value' => '|arg1:skip_vlans|',
+		'max_length' => '1024'
+		),
+	'scan_vlans' => array(
+		'method' => 'textarea',
+		'friendly_name' => 'VLAN ID to Scan',
+		'class' => 'textAreaNotes',
+		'textarea_rows' => '5',
+		'textarea_cols' => '80',
+		'description' => 'VLAN ID to be scaned',
```
Comment (TheWitness) [inline setup.php]: Ditto on previous two comments.

**Review by TheWitness — CHANGES_REQUESTED** [review]
Generally column additions require a version increase.  Just increment the point release in the information file.

File: setup.php (hunk 3 above)
Comment (xxconn) [reply setup.php]: Modified the new field's description.

**Comment by xxconn** [comment]
I will modify INFO and add new fields to database.php tomorrow.

**Comment by xxconn** [comment]
Modified database.php and INFO.

**Review by TheWitness — APPROVED** [review]

**Pull request merged**